src/controllers/message.controller.js

```diff
@@ -0,0 +1,25 @@
+import { Message } from '../models/Message.model.js';
+import { EventEmitter } from 'events';
+
+export const emitter = new EventEmitter();
+
+export const getMessages = async (req, res) => {
+  const whereClauses = {};
+  const { roomId } = req.query;
+  whereClauses.roomId = roomId;
+  res.send(await Message.findAll({ where: whereClauses }));
+};
+
+export const postMessage = async (req, res) => {
+  const { text, author, roomId } = req.body;
+  const message = {
+    text,
+    time: new Date().toString(),
+    author,
+    roomId,
+  };
+
+  await Message.create(message);
+  emitter.emit('message', message);
+  res.status(201).send(message);
+};
```

Comment:
In the `postMessage` function, you should validate the input data before using it. If the client sends invalid or malformed data, it could lead to unexpected behavior or errors. Consider using a library like `joi` to validate the `req.body` data before using it. Also, similar to the previous comment, consider adding error handling for the `Message.create` operation.